src/lib/modules/blockchain_process/proxy.js

```diff
@@ -13,16 +13,29 @@ const utils = require('../../utils/utils');
 const WebSocket = require('ws');
 const WsParser = require('simples/lib/parsers/ws');
 
+const hex = (n) => {
+  let _n = n.toString(16);
+  return _n.length === 1 ? '0' + _n : _n;
+};
+
 const parseJsonMaybe = (string) => {
   let object;
   try {
-    if (string && typeof string === 'string') object = JSON.parse(string);
+    if (typeof string === 'string') {
+      if (string) {
+        object = JSON.parse(string);
+      } else {
+        console.error('Expected a non-empty string');
```

Comment:
This error will be quite confusing if it does appear. It's not clear where it comes from and what triggered it. Could you add something like "Proxy: Expected..."